configure

```diff
@@ -217,12 +217,25 @@ install_local_rebar() {
     fi
 }
 
+install_local_emilio() {
```

Comment:
You need to add all of this to `configure.ps1` too. Are you up to the challenge? 🥊

Reply:
This is a challenge indeed.

Reply:
Here's the diff, feel free to add to this PR:

```diff
diff --git a/configure.ps1 b/configure.ps1
index c74fbcf41..21c079f67 100644
--- a/configure.ps1
+++ b/configure.ps1
@@ -205,6 +205,20 @@ if ((Get-Command "rebar.cmd" -ErrorAction SilentlyContinue) -eq $null)
    $env:Path += ";$rootdir\bin"
 }

+# check for emilio; if not found, get it, build it, add it to path
+if ((Get-Command "emilio.cmd" -ErrorAction SilentlyContinue) -eq $null)
+{
+   Write-Verbose "==> emilio.cmd not found; bootstrapping..."
+   if (-Not (Test-Path "src\emilio"))
+   {
+      git clone --depth 1 https://github.com/wohali/emilio $rootdir\src\emilio
+   }
+   cmd /c "cd $rootdir\src\emilio && rebar compile escriptize"
+   cp $rootdir\src\emilio\emilio $rootdir\bin\emilio
+   cp $rootdir\src\emilio\bin\emilio.cmd $rootdir\bin\emilio.cmd
+   cmd /c "cd $rootdir\src\emilio && rebar clean"
+}
+
 # only update dependencies, when we are not in a release tarball
 if ( (Test-Path .git -PathType Container) -and (-not $SkipDeps) ) {
     Write-Verbose "==> updating dependencies"
```

Note this depends on https://github.com/cloudant-labs/emilio/pull/3 being merged.

Reply:
Thank you very much.